src/libostree/ostree-repo-commit.c

```diff
@@ -4137,6 +4288,13 @@ _ostree_repo_import_object (OstreeRepo           *self,
         return FALSE;
     }
 
+ exit:
```

Comment:
Then this one can be put under the `if (has_object)` conditional above and we can avoid a goto?  (Trying to reduce the number of `goto`s here)